### ISSUE TITLE
nvim-lsp "gather()" failed

### ISSUE THREAD

**Issue by creativenull** [Shougo/ddc-source-nvim-lsp#42]
I am getting the following error as the title says.

<img width="405" alt="image" src="https://github.com/Shougo/ddc-source-nvim-lsp/assets/3767728/5b965c3a-5b7d-40e0-bb1d-4bcbc44b6e1a">

I have not made any changes on my config for a couple months now: https://github.com/creativenull/dotfiles/blob/d5a03647d69f965c0e4d1b80145bcee56215a403/config/nvim/lua/user/ddc.lua#L87

I also turned on debug mode on denops but do not see any additional errors other than the image above.

What else can I try to debug this issue?

Specifications:

Deno v1.35
Denops (commit: https://github.com/vim-denops/denops.vim/tree/fece0a2b24655e42a3fd10dbbf2962d39b381e90)
ddc.vim (commit: https://github.com/Shougo/ddc.vim/tree/6c1de37d138d376326c9bbc1ed0b66d6a244189a)
Neovim 0.9.1

**Comment by Shougo** [comment]
Can you create the minimal vimrc?

**Comment by creativenull** [comment]
I tried creating a minimal vimrc for this but it was working as intended on the minimal file.

So I went back into my config to see what else could've been a problem.

After some digging around I found out that somehow [denops-shared-server](https://github.com/vim-denops/denops-shared-server.vim) plugin needed a restart, this was discovered accidentally when I added it on the minimal vimrc which triggered an install of the shared server. Most likely plugins were cached on the shared server, is my guess.

I will close this issue for now, unless I am able to reproduce it in the future.
